src/slacko.ml

```diff
@@ -599,16 +599,16 @@ let optionally_add key value uri = match value with
 
 let definitely_add key value = optionally_add key (Some value)
 
-let unauthed_endpoint ~base_url e =
+let unauthed_endpoint ~base_url meth =
```

Comment:
Works for me. I don't like trailing underscores particularly, but the apostrophe is fine.